examples/docker/plasticc-on-omnisci/plasticc-omnisci.dockerfile

```diff
@@ -49,21 +46,12 @@ RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge "numpy<1.20.0" omniscidbe4py "ray-core>=1.0" \
-        "ray-autoscaler>=1.0" pandas==1.1.5 cloudpickle==1.4.1 rpyc==4.1.5 "dask>=2.12.0" && \
-    conda install -c intel/label/modin -c conda-forge modin==0.8.3
-
-RUN conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge -c intel ${add_extra_channel} \
-        "daal4py>=2021.1" dpcpp_cpp_rt && \
-    conda install -c conda-forge scikit-learn==0.23.2 xgboost && \
+    conda install -c intel/label/modin -c conda-forge -c intel \
+        modin "ray>=1.0.0" "numpy<1.20.0" "scikit-learn<0.24.0" "daal4py>=2021.1" dpcpp_cpp_rt xgboost && \
```

Comment:
OmniSci supports `numpy==1.20.0`. Can you remove the limitation?

Reply:
Ok I set version to ==1.20.0